File: app/src/main/java/com/drivingschoolrwandaapp/database/entities/SubscriptionPlan.java
package com.drivingschoolrwandaapp.database.entities;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "subscription_plans")
public class SubscriptionPlan {

    @PrimaryKey
    private int id;
    private String planName;
    private String amount;
    private int duration;

    // Getters and setters

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getPlanName() {
        return planName;
    }

    public void setPlanName(String planName) {
        this.planName = planName;
    }

    public String getAmount() {
        return amount;
    }

    public void setAmount(String amount) {
        this.amount = amount;
    }

    public int getDuration() {
        return duration;
    }

    public void setDuration(int duration) {
        this.duration = duration;
    }
}

File: app/src/main/java/com/drivingschoolrwandaapp/database/entities/SubscriptionPlan.kt
package com.drivingschoolrwandaapp.database.entities

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "subscription_plans")
data class SubscriptionPlan(
    @PrimaryKey var id: Int = 0,
    var planName: String? = null,
    var amount: String? = null,
    var duration: Int = 0
)
